_episodes/06-ignore.md

```diff
@@ -22,7 +22,7 @@ $ touch a.dat b.dat c.dat results/a.out results/b.out
 ~~~
 {: .bash}
 
-and see what Git says:
+and after initializing Git, see what Git says:
```

Comment:
I'm -1 on this one. For me "initializing web browser" means open a new window where my web browser will run and I can't do it with Git (command line).